tunnel.go

```diff
@@ -137,6 +130,46 @@ func NewTunnel(account *nkn.Account, identifier, from, to string, tuna bool, con
 	return t, nil
 }
 
+// NewTunnel creates a Tunnel client with given options.
+func NewTunnel(account *nkn.Account, identifier, from, to string, tuna bool, config *Config) (*Tunnel, error) {
```

Comment:
It's less redundant to call `NewTunnels` here.

Reply:
Yes, modified.